brush/src/components/ObjectsTable.js

```diff
@@ -0,0 +1,73 @@
+import React, { useContext } from 'react'
+import PropTypes from 'prop-types'
+import { Table } from 'former-kit'
+import scissors from 'scissors'
+import { defaultTo } from 'ramda'
+import VisionContext from '../context/VisionContext'
+
+const ObjectsTable = ({ onRowClickHandle }) => {
+  const { oam } = useContext(VisionContext).vision
+
+  const rows = oam
+    .map(i => i.data)
```

Comment:
this variable `i` could have a better name like
```suggestion
    .map(row => row.data)
```